ministryofjustice/correspondence_tool_staff
# CT-3057 humanized subject_type, status report in open cases report for Offender SAR

## Description

PR for 
- Humanizing the columns for subject_type and status for open-cases-report for Offender SAR
- Minor bug fix for correspondence_types reseeding script 

## Self-review checklist


### Screenshots

Before 
<img width="1018" alt="Screenshot 2020-10-06 at 14 30 09" src="https://user-images.githubusercontent.com/26111671/95208095-904d8400-07e0-11eb-9a77-908e4d823d3c.png">
After 
<img width="1009" alt="Screenshot 2020-10-06 at 14 30 36" src="https://user-images.githubusercontent.com/26111671/95208106-96436500-07e0-11eb-8636-18aedda320c8.png">

### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3057
### Deployment


### Manual testing instructions

1 - login as Branson user
2 - Click "Download cases" link from "All open cases" tab
3 - Check the columns of subject type and Case status columns
